src/store/participantsStore.js

```diff
@@ -78,6 +81,27 @@ const getters = {
 		return []
 	},
 
+	/**
+	 * Gets the participants array filtered to include only those that are
+	 * currently typing.
+	 *
+	 * @param {object} state - the state object.
+	 * @param {object} getters - the getters object.
+	 * @return {Array} the participants array (if there are participants in the
+	 * store).
+	 */
+	participantsListTyping: (state, getters) => (token) => {
+		return getters.participantsList(token).filter(attendee => {
+			for (const sessionId of state.typing[token]) {
```

Comment:
`state.typing[token]` is not iterable here so we could wrap it with a getter, like we have for `participantsList`

Reply:
That is unexpected, I have manually tested with the latest version (that I have just pushed) and it worked fine. I do not know if it was due to a bug in another part of the code that is already fixed or if the issue is still there in the latest code but I did not hit it :thinking:

Reply:
I will let you know tomorrow, if it's still an issue